framework/src/application/application.js

```diff
@@ -431,6 +431,16 @@ class Application {
 				'rebuild',
 				'processor:sync',
 				'processor:broadcast',
+				'processor:newBlock',
+				'getAccount',
```

Comment:
this is for the `get***` are not the events, so it should be removed from here.
Also, `processor:newBlock` should be removed